src/app/messages/createIntl.ts

```diff
@@ -0,0 +1,27 @@
+import { createIntl, createIntlCache } from 'react-intl';
+import Sentry from '../../helpers/Sentry';
+import enMessages from '../messages/en/index';
+
+export default () => {
+  return {
+    getIntlObject: async(locale: string) => {
```

Comment:
i think i am with you that this doesn't have to be in services anymore. you can just export a function for `createIntl` and call it from anywhere, but its important that we not be re-loading the messages all the time, so you can check to see if the module system caches it, i can't remember if it does that automatically for inline imports. if it doesn't cache it by default you can wrap the whole definition in `lodash/fp/memoize`